product_category_usage_group/tests/test_module.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2018 - Today: GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.exceptions import ValidationError
+from odoo.tests import common
+
+
+class Tests(common.TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

Comment:
FYI you can also use `BaseCommon` class from `odoo.addons.base.tests` to avoid this line